manim/mobject/functions.py

```diff
@@ -62,6 +66,28 @@ def construct(self):
                 self.add(axes, curve1)
                 self.set_camera_orientation(phi=80 * DEGREES, theta=-60 * DEGREES)
                 self.wait()
+
+    .. attention::
+        In case your function contains discontinuous points, observe
```

Comment:
I think this wording is a little strange. How about "If your function has discontinuities, you'll have to specify the location of the discontinuities manually. See the following example for guidance."